### PR TITLE
Added Pocket Casts

### PR DESCRIPTION
**Pull request by katzrkool**
There is no specific url or help article about account deletion unfortunately, so I just put the main support page instead.

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -4878,6 +4878,14 @@
         ]
     },
 
+    {
+        "name": "Pocket Casts",
+        "url": "https://support.pocketcasts.com/",
+        "difficulty": "hard",
+        "notes": "If the web player was purchased, access to it will be lost as well",
```
Comment (Nightfirecat) [inline _data/sites.json]: This should state that account deletion requires an email in addition to the information already here.

**Comment by katzrkool** [comment]
Added the part about a required email

**Comment by Nightfirecat** [comment]
I reworded this slightly and added this in cedb2515dfca01de8896bb8c8b09fb1e188c9b6c.

**Comment by Nightfirecat** [comment]
Thanks for your contribution!

**Pull request closed without merging**